writing-docs/INDEXING.md

```diff
@@ -0,0 +1,75 @@
+# Search indexing rules
+
+### Synopsis
+
+A term or a location in a Pod6 document can be indexed so that it is
+possible to create a named reference pointing to it. This can be used
+to generate a reference in the rendered version of the documentation or
+implement search function on the website hosting the rendered documentation.
+
+### Format
+
+An index item is created using the Pod formatting code `X` in these formats:
+
+    # the `text` is rendered, an item with text `item` under the category `category`
+    # is added to the index
+    X<text|category,item>
+    # can index more than one item
+    X<text|category1,item1;category2,item2;...>
+    # can omit text, nothing is rendered, but an index item will be created
```

Comment:
*and* and anchor. If you are going to delete the "Invisible section anchors" above, at least keep the information somewhere.